mux/node.go

```diff
@@ -84,18 +84,16 @@ type staticNode struct {
 func (n *staticNode) Match(path string) (Node, middleware.Middleware, context.Params, string) {
 	nameLength := len(n.name)
 	pathLength := len(path)
-
-	if pathLength >= nameLength && n.name == path[:nameLength] {
+	if pathLength >= nameLength && n.name == path[:nameLength] || regexp.MustCompile(`{|}`).MatchString(n.name) {
```

Comment:
`staticNode` is not a `regexpNode` and should be not aware of a regex

please revert

Reply:
This was only to detect if the node name contains curly braces as a way to detect if it is a parameter not assuming it's a regexpNode.